src/handlers/network/networkFeeInverseSet.ts

```diff
@@ -0,0 +1,37 @@
+import { mutate, query } from '~amplifyClient';
+import { ContractEvent } from '~types';
+import { output } from '~utils';
+
+export default async (event: ContractEvent): Promise<void> => {
+  const { feeInverse } = event.args;
+  const convertedFee = feeInverse.toString();
+
+  const { items: networkInverseFees } = await query(
+    'getCurrentNetworkInverseFee',
+  );
+
+  const hasExistingNetworkInverseFee = !!networkInverseFees?.length;
+
+  if (hasExistingNetworkInverseFee) {
+    await mutate(
+        'updateCurrentNetworkInverseFee',
+      {
+        input: {
+          id: networkInverseFees[0].id,
+          convertedFee,
+        },
+      },
+    );
+  } else {
+    await mutate(
```

Comment:
Thinking about it now, we could do some optimisation here.
Since there's only one inverse fee ever, you could assign it a constant id. If as you mentioned, the initial network fee entry gets created in tracking function, you don't need the query to see if it exists or get its id.

Reply:
NIce! I'll try it